bramkouwen/Rock-Paper-Scissors-Game
# Building out looping functionality

## What does this PR do?
* It updates the game.py file to contain more functionality

## How do I check this PR?
* Pull down the branch 
* See changes to game.py 

## Screenshots 
None.
